plugins/content/caption/base/lib/kCopyCaptionsFlowManager.php

```diff
@@ -194,4 +195,19 @@ protected static function getClipAttributesFromEntry(BaseObject $object) {
 		}
 		return null;
 	}
+
+	/**
+	 * @param BaseObject $object
+	 * @return bool
+	 */
+	protected static function getClipConcatFlow( BaseObject $object ) {

```

Comment:
rename , this should be isClipConcatFlow

Reply:
done